engine/channels.go

```diff
@@ -156,23 +157,23 @@ func IsClusterChannelID(channelID string) (string, bool) {
 
 // FullyQualifiedChannelName returns the unique channel name made up of channel name string suffixed with root block id
 // The root block id is used to prevent cross talks between nodes on different sporks
-func FullyQualifiedChannelName(channelID string, rootBlockID string) string {
+func FullyQualifiedChannelName(channel network.Channel, rootBlockID string) string {
```

Comment:
We could rename this function to `TopicFromChannel`. It is used to suffix the channel name with the root block id.
If you look at all the usages of this function, the callers store the return value into a variable called `topic` :)
That way we can also get rid of this notion of `FullyQualifiedChannelName` and only gave `channels` at the upper layer and `topic` at the lower layer.

Reply:
Defined `Topic` as a type, and applied the comment.